h2o-py/h2o/model/model_base.py

```diff
@@ -447,6 +459,8 @@ def show(self):
             model["scoring_history"].show()
         if "variable_importances" in model and model["variable_importances"]:
             model["variable_importances"].show()
+        if "model_parameters" in model and model["model_parameters"]:
```

Comment:
We only want to make them available, not print them explicitely, as the amout of parameters could be quite large. WDYT ?

Reply:
@Pscheidl By default TwoDimTable will be printed in a truncated fashion - 5 top rows and 5 last rows. Don't think it is too much. It could serve as a hint that parameters are exposed and user can go a get whole list.

Reply:
Good enough to me.